openvdb/Makefile

```diff
@@ -62,6 +62,8 @@
 #                       (default: link against shared libraries)
 #   debug=yes           build with debugging symbols and without optimization
 #   verbose=yes         run commands (e.g., doxygen) in verbose mode
+#   warnings=yes        Enable a collection of pre defined compiler warnings
```

Comment:
"`strict`" would be a better name for this option, since all compilers generate some warnings even without any flags.

Reply:
done